picki/app/events/events.css

```diff
@@ -65,78 +41,45 @@ h1 {
   margin: 0 0 25px 0;
 }
 
-/* Form elements */
-.form-group {
-  margin-bottom: 25px;
-}
-
-.form-label {
-  font-size: 16px;
-  font-weight: 600;
-  margin-bottom: 10px;
-  color: #363636;
+/* Event name input */
+.event-name-input{
+    margin-top: 50px;
+    margin-bottom: 50px;
 }
 
 .event-input {
   width: 100%;
-  padding: 15px;
-  border-radius: 25px;
-  border: 1px solid #e1e1e1;
+  padding: 12px 15px;
+  border-radius: 33px;
+  border: 1px solid #cccccc;
   font-size: 16px;
   box-sizing: border-box;
   outline: none;
+  color: #363636;
 }
 
-/* Budget slider */
-.budget-indicator {
-  display: flex;
-  align-items: center;
-  margin-bottom: 10px;
-}
-
-.budget-icon {
-  width: 24px;
-  height: 24px;
-  border-radius: 50%;
-  background-color: #5e17eb;
-  color: white;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: 14px;
-  font-weight: bold;
+.event-input::placeholder {
+  color: #cccccc;
+    font-size: 16px;
 }
 
-.budget-slider {
-  width: 100%;
-  height: 6px;
-  -webkit-appearance: none;
-  appearance: none;
-  background: #e1e1e1;
-  border-radius: 3px;
-  outline: none;
+/*Budget slider */
+.budget-slider-label {
```

Comment:
.buget-slider-label and other labels look like they ahve the same styling, please combine these into a single class.

Reply:
Perhaps <label> can be it's own component?